pkg/console/operator/operator.go

```diff
@@ -235,21 +235,8 @@ func (c *consoleOperator) handleSync(configs configSet) error {
 		return fmt.Errorf("console is in an unknown state: %v", updatedStatus.Spec.ManagementState)
 	}
 
-	err := c.sync_v400(updatedStatus, configs)
-
-	c.HandleDegraded(updatedStatus, "SyncError", func() error {
-		if !customerrors.IsSyncError(err) {
-			return err
-		}
-		return nil
-	}())
-
-	c.HandleDegraded(updatedStatus, "CustomLogo", func() error {
-		if !customerrors.IsCustomLogoError(err) {
-			return err
-		}
-		return nil
-	}())
+	// TODO: Do we no longer need the err here since we are handling all the conditions down 1 layer?
+	_ = c.sync_v400(updatedStatus, configs)
```

Comment:
since we dont need the `error` anymore can we remove the return value from the `sync_400` func ?
```
func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console, set configSet) {
```

Or thats the part of the TODO ?